lib/components/base-components/NormalComponent.ts

```diff
@@ -639,6 +634,23 @@ export class NormalComponent<
     return this._parsedProps.schPortArrangement
   }
 
+  _getNormalizedPinLabels(): string[][] {
```

Comment:
```suggestion
  _getNormalizedPinLabelsFromSchPortArrangement(): string[][] {
```

Reply:
i think otherwise this is a bit confusing because it doesn't look at the footprint or other sorts of pin labels (even in a later phase it could look at all the initialized ports, so depending on the context it's unclear what the source is)